packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -314,11 +345,11 @@ export interface JsonArrayNodeSchema extends JsonNodeSchemaBase<NodeKind.Array,
 }
 
 // @alpha
-export type JsonCompatible = string | number | boolean | null | JsonCompatible[] | JsonCompatibleObject;
+export type JsonCompatible<TExtra = never> = string | number | boolean | null | JsonCompatible<TExtra>[] | JsonCompatibleObject<TExtra> | TExtra;
```

Comment:
Probably better to put questions about APIs on their definitions where you can see the documentation for them rather than in the API report.

That said it was missing docs on the new option, so I have added them.